package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -151,14 +151,16 @@ preload_rke2_images()
     RKE2_IMAGES_LISTS_DIR="/tmp/images-lists"
     mkdir -p $TARGET/$RKE2_IMAGES_DIR
     mkdir -p $TARGET/$RKE2_IMAGES_LISTS_DIR
-    mount --bind ${ISOMNT}/bundle/harvester/images $TARGET/$RKE2_IMAGES_DIR
     mount --bind ${ISOMNT}/bundle/harvester/images-lists $TARGET/$RKE2_IMAGES_LISTS_DIR
 
+    echo "Copying RKE2 images to the target location..."
+    rsync -ah --progress ${ISOMNT}/bundle/harvester/images/rke2-images.*.tar.zst $TARGET/$RKE2_IMAGES_DIR
```

Comment:
would it be better to specify `*.tar.zst` only?

Reply:
There are actually no differences between "copy rke2-images first" and "copy all images." `rsync` will only copy files that are missing at the destination. The reason we only copy rke2-images here is to make containerd available quicker and to make the whole process looks more responsive; therefore, we have a two-stage copy.